interface-definitions/interfaces_wireless.xml.in

```diff
@@ -428,6 +416,133 @@
                   </leafNode>
                 </children>
               </node>
+              <leafNode name="require-vht">
+                <properties>
+                  <help>Require stations to support VHT PHY</help>
+                  <valueless/>
+                </properties>
+              </leafNode>
+              <node name="he">
+                <properties>
+                  <help>HE (High Efficiency) settings</help>
+                </properties>
+                <children>
+                  <leafNode name="channel-set-width">
```

Comment:
Maybe it will be better this way?
```
channel-width
```

Reply:
I took example from options already present in HT and VHT sections, lines [45](https://github.com/vyos/vyos-1x/blob/4ff64d4a73e84cacad1db3029854d846f94b1b6e/interface-definitions/interfaces_wireless.xml.in#L45) and [275](https://github.com/vyos/vyos-1x/blob/4ff64d4a73e84cacad1db3029854d846f94b1b6e/interface-definitions/interfaces_wireless.xml.in#L275) which had been around like that long time. If we changed it on one spot, we should change it on all spots and this would break CLI against existing configurations, requiring a mitigation script. Therefore, I went with what was already in HT and VHT sections.